sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -156,15 +157,18 @@ private void saveSample(SampleCreateForm createForm) {
 			pathogenTest.setTestedDisease((Disease) (createForm.getField(PathogenTestDto.TESTED_DISEASE)).getValue());
 			pathogenTest.setTestDateTime((Date) (createForm.getField(PathogenTestDto.TEST_DATE_TIME)).getValue());
 			pathogenTest.setTestResultText((String) (createForm.getField(PathogenTestDto.TEST_RESULT_TEXT)).getValue());
-			newSample.setPathogenTestResult(testResult);
 			FacadeProvider.getSampleFacade().saveSample(newSample);
 			FacadeProvider.getPathogenTestFacade().savePathogenTest(pathogenTest);
-
 			final EventParticipantReferenceDto eventParticipantRef = newSample.getAssociatedEventParticipant();
+			EventParticipantDto eventParticipant =
```

Comment:
I think we need a null check for the eventParticipantRef here. E.g. saving a sample that is only related to a case would not have an event participant.